book/deps.sh

```diff
@@ -0,0 +1,20 @@
+# Basic install script for pdf generation dependencies on linux, works on debian
+
+# Install pandoc, we use version 2.11.2
+wget https://github.com/jgm/pandoc/releases/download/2.11.2/pandoc-2.11.2-1-amd64.deb
+sudo dpkg -i pandoc-2.11.2-1-amd64.deb
+
+# Install tinytex
+wget -qO- "https://yihui.org/tinytex/install-bin-unix.sh" | sh
+
+# Install needed latex packages for our use
+tlmgr install \
+  environ \
```

Comment:
Nice! Curious how you managed to _decrease_ the dependencies here from the previous larger list of deps.

Reply:
Previously we used some 3rd-party template with lots of dependencies, which we hadn't actually used. This time we use the standard template with some minor additions (e.g. `tcolorbox`, `fancyhdr`, etc)